pkgs/by-name/pr/prismlauncher/package.nix

```diff
@@ -55,7 +55,7 @@ let
 in
 
 symlinkJoin {
-  name = "prismlauncher-${prismlauncher'.version}";
+  inherit (prismlauncher') pname version;
```

Comment:
`pname` shouldn't be inherited here, as the unwrapped derivation's pname includes `-unwrapped`